content/language-agnostic/encyclopedia/what-is-a-parameter-and-an-argument?.md

```diff
@@ -0,0 +1,28 @@
+<!--
+---
+Subjects:
+  - "What is a parameter and an argument?"
+Tags: 
+  - "language-agnostic"
+  - "functions"
+Catalog Content:  # If listing multiple URLs, please put the most relevant one first 
+  - "https://www.codecademy.com/courses/introduction-to-javascript/lessons/functions/exercises/parameters"
+---
+-->
+
+# What is a parameter and an argument?
+
+A parameter is the name of a variable passed into a function. Parameters allow functions to accept inputs. An argument, on the other hand, is the actual value of the variable (aka. the parameter) passed into a function.
+
+
+## Example with Kotlin
```

Comment:
let's take out "with Kotlin"

for the encyclopedia entries, let's try to keep them language-agnostic if possible